src/shared/components/data-table/components/table-controls/components/pagination/Pagination.tsx

```diff
@@ -22,18 +22,21 @@ import { TableContext } from 'src/shared/components/data-table/DataTable';
 import styles from './Pagination.scss';
```

Comment:
A general question I meant to ask in the previous DataTable PR.

At the moment, the Pagination component is tightly coupled with the DataTable component. Should we try to ask Andrea whether she expects the pagination element in any other context in her views; and if she does, would that be grounds for extracting Pagination into a shared component?

Reply:
I think it will be fairly easily move it to be a shared component. Is that something we need to think about doing now?

Reply:
It's just that I am starting to see the Pagination component being more and more entangled with the table. It starts looking into the table context, etc. If we expect the Pagination component to reappear, we may want to separate the UI component early. Doesn't have to be this specific PR, but still.

Reply:
I agree that it is currently entangled but like I said, it should be easy to pass the data it needs as props. I can do that in a separate PR.

Reply:
I asked Andrea; and she agreed that the pagination component is likely to reappear in Global Search; and when it does, it should look the same as in the table. So it does sound like we have a strong case for a shared component. Could be in this PR on in a separate one.